app/experimenter/experiments/models.py

```diff
@@ -574,6 +582,17 @@ def should_have_population_percent(self):
     def should_have_total_enrolled(self):
         return self.type not in (self.TYPE_GENERIC, self.TYPE_ROLLOUT)
 
+    @property
+    def display_platforms(self):
+        if {
+            ExperimentConstants.PLATFORM_MAC,
+            ExperimentConstants.PLATFORM_LINUX,
+            ExperimentConstants.PLATFORM_WINDOWS,
```

Comment:
Also if you move the list to a constant then you can just refer to the constant here too